.github/workflows/cmake.yml

```diff
@@ -0,0 +1,27 @@
+name: cmake
+
+on:
+  push: {}
+  pull_request: {}
+
+jobs:
+  job:
+    name: cmake.${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
and windows please :)

https://github.com/google/benchmark/blob/main/.github/workflows/build-and-test.yml is the full cmake CI  action.